app/javascript/packages/document-capture/components/submission-complete.jsx

```diff
@@ -48,12 +48,7 @@ function SubmissionComplete({ resource }) {
     return () => window.clearTimeout(sleepTimeout.current);
   }
 
-  return (
-    <>
-      <CallbackOnMount onMount={handleResponse} />
-      <SubmissionInterstitial />
```

Comment:
We need to remove the import statement for this at the top of the file, since it's no longer used.